pom.xml

```diff
@@ -1200,6 +1188,7 @@
                 <module>vaadin</module>
                 <module>libraries-3</module>
                 <module>web-modules</module>
+                <module>jenkins-modules</module>
```

Comment:
we can't move jenkins-module to jdk9+ profile, we attempted in https://team.baeldung.com/browse/JAVA-19670 and it failed